types/iobroker/index.d.ts

```diff
@@ -1008,6 +1047,47 @@ declare global {
             /** Deletes a binary state from the states DB */
             delBinaryStateAsync(id: string, options?: unknown): Promise<void>;
```

Comment:
the non-foreign methods are deprecated until 4.1 because we have to migrate all to foreign to add the fixId here.. it's a bit of a mess, maybe remove the typings for now, and add them back? I am quite unsure how to best proceed here

Reply:
will deprecate for now